components/src/core/drawer/drawer-body/nav-item/drawer-nav-item.component.scss

```diff
@@ -110,25 +112,30 @@ pxb-drawer-nav-item {
         z-index: 1;
         display: flex;
         flex-direction: column;
+        .mat-icon {
+            font-size: 1.5rem;
+            height: 1.5rem;
+            width: 1.5rem;
+        }
     }
     .pxb-drawer-nav-item-rail {
         z-index: 1;
         display: flex;
         flex-direction: column;
-        min-height: 72px;
-        width: 72px;
+        min-height: 4.5rem;
+        width: 4.5rem;
```

Comment:
Looks like the Nav Rail (default) has a 4.5rem width.  We adjusted the condensed/closed-persistent rail width to be calc(1.5rem + 16px * 2), so we might have to tweak this a bit.

Reply:
I've updated the width here to be 3.5rem + 16px.